style.css

```diff
@@ -504,7 +744,371 @@ ul {
   font-size: 17px;
   font-weight: 400;
   line-height: 20px;
-  letter-spacing: 0.01em;
   margin-bottom: 20px;
   color: #172b4d;
 }
+
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .logo {
+    font-size: 18px;
+    font-weight: 850;
+    line-height: 20px;
+    background-color: transparent;
+    margin-left: 100px;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger {
+    display: none;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 70px;
+    gap: 15%;
+  }
+
+  .menu-item a {
+    display: flex;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
```

Comment:
### Sorry For putting too much changes requests in this section, I didn't find a better approach, if you find any unclear comments just tag me in the PR thread and point out your concerns so I can answer them 👍🏼 

#### The headline section can be improved since it has issues that we need to work on 💪🏼
 
- [ ] At some screen sizes the headline background loses its flow and starts diverting from the design that we suppose to duplicate from the Figma design 💯, my suggestion is to set the background using [CSS Multiple Backgrounds](https://www.w3schools.com/css/css3_backgrounds.asp) after exporting each asset that compounds the background images by its own, and then set each asset at its own positioning 💪🏼, **NOTE❗❗❗: the assets are highlighted in blue squares in the bellow screenshot from Figma **
- [ ] Both spacing from the left and right sides of the headline section content should be the same, kindly ensure that you use the same value for `margin-left` and `margin-right` for the headline.

| Your Design | Figma Design |
|---|---|
| ![image](https://user-images.githubusercontent.com/68030297/230694793-97fc373b-fe6b-4f2c-b588-f69dd06f36db.png) | ![image](https://user-images.githubusercontent.com/68030297/230694878-ab51b0e8-78f5-44fc-b948-3025b81b3631.png) |